### ISSUE TITLE
Quotas admin panel missing from side selection

### ISSUE THREAD

**Issue by innovate-invent** [galaxyproject/galaxy#11594]
Galaxy 21.01
As you can see in the photo there is no "Quotas" option other than on the admin landing page:
![image](https://user-images.githubusercontent.com/1768820/110735714-2c8c1480-81df-11eb-8eb8-811be2f1d0b5.png)

**Comment by martenson** [comment]
Have you enabled quotas in the Galaxy configuration?

**Comment by innovate-invent** [comment]
Right... I guess the issue is the reverse. The Quotas link should be removed from (or greyed out with a note to enable) the admin landing page.
